src/components/AddProfile.js

```diff
@@ -207,9 +210,10 @@ class AddProfile extends React.Component {
                       </Form.Label>
                       <Col sm="10">
                         <Form.Control
-                          required
+                          required = {(this.state.disabledRefferedBy)? "true" : "false"}
```

Comment:
I think true and false should be boolean

Reply:
fixed